gui/qt/paytoedit.py

```diff
@@ -266,23 +266,24 @@ def textUnderCursor(self):
         return tc.selectedText().strip()
 
     def keyPressEvent(self, e):
-        if self.isReadOnly():
+        if self.isReadOnly() or not self.hasFocus():
+            e.ignore()
```

Comment:
Just for my own edification: it's possible to receive the even even if not hasFocus() if a child widget ignores the event right? I seem to remember that's how it works hence how we ended up here...

Reply:
Yea, that is my understanding as well. Note that you have to explicitly call .ignore() on the event in the child as accepting it is the default.